### DISCUSSION TITLE
After deploying the kube-bench to the cluster as deployment, obtaining the binary file failed

### DISCUSSION THREAD

**Discussion by Erica177** [aquasecurity/kube-bench#1293] [Questions and Help]
I am tentatively deploying the kube bench as a deployment in the cluster and controlling its output. But after deployment, I found the function psFunc in the log output of the pod reported an error: [/bin/ps - C etcd - o cmd -- no headers]: exit status 1

Later, when I entered the container to view it, I found that the process "etcd" in the host could not be found in the container, but I had set hostPID=true
![image](https://user-images.githubusercontent.com/30773688/192994537-29262dba-9b07-4e84-86e0-ce32de9064dc.png)

**Comment by mozillazg** [comment]
Could you please attach the debug logs? (run kube-bench with the flag -v 3)

**Reply by Erica177** [reply]
I deploy the kube bench job on the Alibaba Cloud ack cluster. The output of the container shows that executing  `/bin/ps -C kube apiserver - o cmd -- no headers`  in the container failed, but I can get correct results by executing this command on the host
![image](https://user-images.githubusercontent.com/30773688/193199790-87a81a8a-a2d6-41b2-aca0-3f44409a5927.png)
![image](https://user-images.githubusercontent.com/30773688/193200101-56ef3fd0-90c3-4846-b2fb-a5c322e301f3.png)

**Reply by mozillazg** [reply]
Does the job running on the master node? Could you please try again using https://github.com/aquasecurity/kube-bench/blob/main/job-master.yaml (changed the command to `["kube-bench", "run", "--targets", "master", "-v", "3"]` before apply)?
